networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -11,7 +11,7 @@ The Kubernetes NMState Operator provides a Kubernetes API for performing state-d
 
 [IMPORTANT]
 ====
-Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, and LinuxONE installations.
+Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, LinuxONE, and VMWare vSphere installations.
```

Comment:
```suggestion
Red Hat supports the Kubernetes NMState Operator in production environments on bare metal, IBM Power, IBM Z, LinuxONE, and VMWare vSphere installations.
```

Reply:
@tmalove In this case, bare-metal is a compound adjective: "bare-metal installations," so the hyphen is correct per the IBM Style Guide.